examples/examples-cpu/nyc-taxi-snowflake/README.md

```diff
@@ -0,0 +1,207 @@
+|<img src="img/taxi.png" width="200" /> | <img src="img/saturn.png" width="400" />|<img src="img/snowflake.png" width="400" /> |
+| -- | -- | -- |
+
+# NYC Taxi analysis with Saturn Cloud and Snowflake
+
+The notebooks in this example showcase a data science workflow with NYC taxi data, executed on [Saturn Cloud](https://www.saturncloud.io/) with a database hosted in [Snowflake](https://www.snowflake.com/). For more information on how to connect to Snowflake using Saturn, Dask, and Pandas, refer to the `snowflake` example.
+
+> **Note:** Running this example requires a Snowflake account. You can see how to set up a [free trial here](https://docs.snowflake.com/en/user-guide/admin-trial-account.html).
+
+This example is a subset of a [larger demo](https://youtu.be/SgXSIbB4Hik), reduced to quickly highlight key features of Saturn Cloud and Snowflake together. The larger demo includes the following:
+
+<img src="img/pipeline.png" width="800">
+
+All code for the full demo is [available here](https://github.com/saturncloud/saturn-cloud-examples/tree/main/taxi_demo). You can follow the instructions there to create a new project and run the full examples. The example you are currently in is a reduced version to quickly (and cost-effectively) highlight key features of Saturn Cloud.
+
+The notebooks in _this_ example cover:
+
+1. Load records from CSV files into Snowflake (Snowflake SQL)
+1. Create and deploy a dashboard with exploratory analysis (`holoviz`, `bokeh`, `panel`, `snowflake-connector-python`)
+1. Train a number of machine learning models using single-node Python tools (`scikit-learn`, `xgboost`)
+1. Use Dask to distribute and speed up model training (`dask-ml`, `dask-xgboost`)*
+1. Deploy models via REST API (`flask`)
+
+_\* These examples illustrate how to launch and utilize a Dask cluster with Saturn._
+
+You are free to open each notebook in this example and start playing around! For a guided experience, follow the steps below.
+
+## Load data into Snowflake
+
+> **Note**: If you ran the `snowflake` example and already loaded the NYC taxi data into your Snowflake account, you can skip this step. This example uses the same data.
+
+First step is to add your Snowflake credentials for notebooks in Saturn to access. To avoid setting credentials directly in notebooks, we recommend uploading credentials stored in a .yml file on the "Credentials page" in Saturn Cloud.
+
+- Type: `File`
+- Shared with: `<your username>`
+- Path: /home/jovyan/snowflake_creds.yml
+- Value: .yml file contents (below)
+
+The .yml file can specify any arguments that can be passed to `snowflake.connector.connect`, such as:
+
+```yaml
+account: ...
+user: ...
+password: ...
+role: ...
+```
```

Comment:
can you link to https://docs.snowflake.com/en/user-guide/python-connector-example.html#connecting-to-snowflake, which describes how to set these? Finding the right value for `account` is not obvious.